src/components/FiatOnrampModal/index.tsx

```diff
@@ -10,7 +11,7 @@ import Circle from '../../assets/images/blue-loader.svg'
 import Modal from '../Modal'
 
 const Wrapper = styled.div`
-  background-color: ${({ theme }) => theme.backgroundSurface};
+  background-color: ${({ theme }) => theme.white};
```

Comment:
super tiny design nit unrelated to analytics (obv)

Reply:
moonpay iframe content is always white, so in dark mode the wrapper was creating a thick `backgroundSurface` border around the iframe content